examples/gw_incompressible_3d.py

```diff
@@ -0,0 +1,162 @@
+from gusto import *
```

Comment:
Good point - it's on a branch! If you don't mind, I'd rather keep sk_nonlinear_3d and drop this one, purely because sk_nonlinear_3d is already set up to fit with the code_restructure whereas I'd have to change this one.